adhocteam/Head-Start-TTADP
# Add sorting to my alerts

**Description of change**
Adds sorting to "My Alerts"

**How to test**
- pull changes
- verify that my alerts can be sorted

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/187

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
